package.json

```diff
@@ -16,12 +16,17 @@
   "scripts": {
     "check-errors": "tsc --noemit",
     "build-publish": "rimraf dist && cti entrypoint ./src -o index.ts && cross-env APP_ENV=production rollup -c",
-    "validate": "npm run test"
+    "validate": "npm run test",
+    "format": "prettier --write \"**/*.{ts,tsx}\""
```

Comment:
fantastic that you've added prettier, could you also add the pre-commit hooks & .prettierrc.json that the https://github.com/XRFoundation/XREngine repo has too? this way it is more standardized across all our projects

Reply:
sure

Reply:
done , updated to v1.1 as well